wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/TrapezoidProfileSubsystem.java

```diff
@@ -90,11 +90,16 @@ public void enable() {
     m_enabled = true;
   }
 
-  /** Disable the TrapezoidProfileSubsystem's output. */
+  /** Disable the TrapezoidProfileSubsystem's output. Called on robot disable. */
   public void disable() {
     m_enabled = false;
   }
 
+  @Override
+  public void disabledInit() {
+    disable();
+  }
+
```

Comment:
Perhaps re-abstract this? Because in any case, subclasses have to override this to stop motors, reset controllers, etc. Teams will have to call `disable()` or super, so "forcing" them with an abstract function might make this easier.

Also, after talking with @calcmogul it seems that the profile recalculation in `periodic` should be moved into the `enabled` conditional. To quote:
> I think the profile should probably stop too. Otherwise, the state is going to run away from you while the controller is disabled, and you'll get a jolt upon reenabling (assuming the motor output also zeroed itself in the subclass while disabled).